Dockerfile

```
@@ -0,0 +1,7 @@
+FROM openjdk:11
+WORKDIR /
+ADD target/*.jar app.jar
+RUN useradd -m myuser
+USER myuser
+EXPOSE 8080
+CMD java -jar -Dspring.profiles.active=prod  app.jar
```

```suggestion
CMD ["java", "-jar", "-Dspring.profiles.active=prod", "app.jar"]
```
[Best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#cmd)

---

done